clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -85,7 +95,15 @@ public synchronized Cluster fetch() {
      * Add the topic to maintain in the metadata
      */
     public synchronized void add(String topic) {
```

Comment:
Would it be a good idea to change the name of `add` and `setTopics` below to reflect the expiration policy?